2d/platformer/src/Actors/Player.gd

```diff
@@ -155,6 +155,8 @@ func _on_coinCollected():
 func _on_orbCollected():
 	self.orbs = self.orbs + 1
 	$UI/amtOrbsLabel.text = "Orbs: " + String(self.orbs)
+	if (self.orbs >= 4):
```

Comment:
Should have a constant global var for the amount of orbs needed.

Reply:
It says the code is outdated for Player.gd, might want to resolve that first.

Reply:
@mbagwell that just means he made a commit to it after I made a comment. In his commit, he created a constant var like I suggested, so it is fixed.